repose-aggregator/components/filters/openstack-identity-v3-filter/src/main/scala/org/openrepose/filters/openstackidentityv3/utilities/OpenStackIdentityV3API.scala

```diff
@@ -43,13 +44,13 @@ import scala.util.{Failure, Random, Success, Try}
 class OpenStackIdentityV3API(config: OpenstackIdentityV3Config, datastore: Datastore, akkaServiceClient: AkkaServiceClient)
   extends LazyLogging {
 
+  import OpenStackIdentityV3API._
+
   private final val SC_TOO_MANY_REQUESTS = 429
   private final val TOKEN_ENDPOINT = "/v3/auth/tokens"
   private final val GROUPS_ENDPOINT = (userId: String) => s"/v3/users/$userId/groups"
 
   private final val ADMIN_TOKEN_KEY = "IDENTITY:V3:ADMIN_TOKEN"
-  private final val TOKEN_KEY_PREFIX = "IDENTITY:V3:TOKEN:"
-  private final val GROUPS_KEY_PREFIX = "IDENTITY:V3:GROUPS:"
```

Comment:
I moved these to be able to access them external to this class, but apparently left things inconsistent. I have a TODO for this somewhere, so I'll fix it.